Assets/MixedRealityToolkit.Providers/UnityARCameraSettings.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Do we want this provider folder name to be more along the lines of `ARFoundation`? Or are we specifically only doing the camera settings here?

Reply:
Only the camera settings... so far. We could make the folder and assembly be "UnityAR"